xml/decoders/esu/v5fnOutCVs.xml

```diff
@@ -651,9 +701,34 @@
     <variable label="Coupler Force" CV="16.0.278" default="31" item="ESU V4 Function Output A1-1 Slider 7" mask="XXXVVVVV">
         <qualifier>
             <variableref>AUX1 [1] Mode</variableref>
-            <relation>eq</relation>
+            <relation>ge</relation>
             <value>28</value>
         </qualifier>
+        <qualifier>
+            <variableref>AUX1 [1] Mode</variableref>
+            <relation>ne</relation>
+            <value>29</value>
+        </qualifier>
```

Comment:
I don't understand the qualifiers near here.  They seem to all reference "AUX1 [1] Mode" and check for >=28, ==29, ==30, ==31, ==32, ==33.  Aren't those last terms redundant?

Reply:
It's actually "check for: (var ge 28) && (var ne 29) && (var ne 30) && (var ne 31) && (var ne 32) && (var le 33)".

This resolves to "check for: (var eq 28) || (var eq 33)". 

Clumsy, but unfortunately there's no OR operator for multiple qualifier elements.

Reply:
Ah, thank you! Now I understand.